aiida/backends/tests/cmdline/commands/test_user.py

```diff
@@ -15,13 +15,13 @@
 from aiida.cmdline.commands import cmd_user
 from aiida import orm
 
-user_1 = {
+user_1 = {  # pylint: disable=invalid-name
```

Comment:
Why not just rename it to a proper constant?

Reply:
fixed.